README.md

```diff
@@ -6,5 +6,16 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework that is written in Ruby that provides an organized approach to
+processing requests to a website backend
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
 1. Describe the flow of a PATCH request as it goes through the app.
+is a command that is used to update an existing record
+-the user would initiate the command and send it to the server
+-the server would determine the correct controller to send the PATCH command to and send it
+-the controller would send the PATCH command to the correct model (there is only one)
```

Comment:
Not quite. The controller is responsible for handling the request, and it has access to certain data and meta-data that came along with the request, but it doesn't actually touch the request itself. It definitely doesn't share it with the model. All the model gets from the controller is a direction to perform some CRUD action.